ginipaybank/src/main/res/layout/gpb_fragment_digital_invoice_info.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/gpb_digital_invoice_onboarding_fragment_background"
+    android:orientation="vertical"
+    tools:context=".digitalinvoice.info.DigitalInvoiceInfoFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1">
+
+        <TextView
+            android:id="@+id/gpb_info_text_1"
```

Comment:
Ids don't have to be prefixed with `gpb_` since they are not customisable resources

Reply:
@sergiuciuperca Could you remove the prefix

Reply:
@AlexGabor they were prefixed before, in the old views also. Should the prefix be removed everywhere?

Reply:
Ideally yes. It was using kotlin synthetics before, but I removed it since it's deprecated. For view binding the ids don't have to be globally unique